app/src/main/res/values/strings.xml

```diff
@@ -589,4 +587,11 @@
     <string name="downloads_storage_use_saf_title">Use SAF</string>
     <string name="downloads_storage_use_saf_summary">The Storage Access Framework allows downloads to an external SD card.\nNote: some devices are not compatible</string>
     <string name="choose_instance_prompt">Choose an instance</string>
+    <string name="videos_fixed">Videos</string>
+    <string name="five_seconds">5 seconds</string>
```

Comment:
Wouldn't be `%s seconds` better here? We would need to add the seek_duration_description items dynamically, but this shouldn't be a problem.

Reply:
I just did it.
The reason why I've chosen this way is: summary is now fine, but when you click on the list, it only shows the value (eg 5, 10, 15…), but not seconds anymore.

<img alt="" src="https://user-images.githubusercontent.com/60391705/73571717-16057c80-4467-11ea-8ee1-3f2c79e85cfd.png" width="200"> <img alt="" src="https://user-images.githubusercontent.com/60391705/73571718-169e1300-4467-11ea-80ff-a74d9e0a9ec8.png" width="200">

Reply:
I firstly done that, but there is a problem: summary is now fine, but when you click on the list, it only shows the value (eg 5, 10, 15…), but not seconds anymore.

<img alt="" src="https://user-images.githubusercontent.com/60391705/73571717-16057c80-4467-11ea-8ee1-3f2c79e85cfd.png" width="200"> <img alt="" src="https://user-images.githubusercontent.com/60391705/73571718-169e1300-4467-11ea-80ff-a74d9e0a9ec8.png" width="200">

(I made this with the last commit)